src/tests/SimTest.cpp

```diff
@@ -129,7 +167,7 @@ SimTest::SimTest() {
             &SimTest::recomputeNavmeshWithStaticObjects,
             &SimTest::loadingObjectTemplates,
             &SimTest::buildingPrimAssetObjectTemplates,
-            &SimTest::addSensorToObject});
+            &SimTest::addSensorToObject}, NumSims );
```

Comment:
```suggestion
            &SimTest::addSensorToObject}, Cr::Containers::arraySize(SimulationBuilder) );
```

and you don't need the `NumSims` variable at all (sorry, it's still like that in the docs, i have to clean that up)

Reply:
[Fixed.](https://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#TestSuite-Tester-instanced). Sorry again, bad decisions from 2014 shouldn't still be preserved seven years later :D